### ISSUE TITLE
loadOpenAL is neither @nogc or nothrow

### ISSUE THREAD

**Issue by JackStouffer** [BindBC/bindbc-openal#4]
```
Error: @nogc function D main cannot call non-@nogc function bindbc.openal.binddynamic.loadOpenAL
Error: function bindbc.openal.binddynamic.loadOpenAL is not nothrow
```

The function is not annotated.

**Comment by mdparker** [comment]
Sorry, Jack. I missed this when you posted it. Looks like I'm missing the attributes on more than the OpenAL loader.

**Comment by mdparker** [comment]
Fixed in 0.5.0.
